recipes/idaes_minimal/meta.yaml

```
@@ -0,0 +1,54 @@
+{% set name = "idaes_minimal" %}
+{% set version = "1.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/IDAES/idaes-pse/archive/bdfddf1d1761bdba408567a83e198d549e8c741d.zip
+  sha256: 99b365817d86b82189912b8ca2c2a0a9b794ff584e3ce2fa7c1c8bb8f0f8787b
+
+build:
+  number: 0
+  skip: true  # [py<35]
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  build:
+    - python
+    - pip
+  host:
+    - python
```

If (and only if) you make this `noarch` you can constrain the Python version with:
```suggestion
    - python >=3.5
```